### PR TITLE
feat(analytics): Log exposure abstraction

### PR DESCRIPTION
**Pull request by denamwangi**
Add abstraction for logging experiment exposures.

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/log_exposure.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import HookStore from 'app/stores/hookStore';
+
+/**
+ * @param {String} name name of the experiment
+ * @param {Object} data Experiment data to be recorded: {unit_name, unit_id, params}
+ * 					note that params need to string
+ */
```
Comment (denamwangi) [inline src/sentry/static/sentry/app/utils/log_exposure.jsx]: still not super clear on why passing an object for params doesn't work- will dig in at some point to figure that out.

File: src/sentry/static/sentry/app/utils/log_exposure.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+export default function log_exposure(name, data) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/log_exposure.jsx]: Can you rename this (+ file) to be `camelCase`?

Comment (denamwangi) [reply src/sentry/static/sentry/app/utils/log_exposure.jsx]: ugh pythonBrain- thanks

**Review by billyvg — APPROVED** [review]
Looooks good!

This is simple enough that a test isn't super necessary, but it wouldn't hurt to have one.

**Pull request merged**